tests/console/rsync.pm

```diff
@@ -40,7 +40,18 @@ sub run {
     if (script_run('! test -e ~/.ssh')) {
         assert_script_run('ssh-keygen -R localhost');
     }
+
+    if (is_opensuse) {
+        my $output = script_output('cat /etc/ssh/sshd_config | grep "#PermitRootLogin"');
+        if ($output =~ 'prohibit-password') {
+            clear_console;
+            assert_script_run("sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config");
```

Comment:
I would just call this line unconditionally as the replace would only take place when root authentication is disabled anyway

Reply:
Updated. The changes are in this PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6190